test/integration/simple-deployment-recursive-task/01-assert.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     keptn.sh/app: waiter
     keptn.sh/version: "0.4"
     keptn.sh/workload: waiter-waiter
+  labels:
```

Comment:
added the labels to better filter for the expected objects in the assertions. For checks against objects where we do not know the name (due to a random suffix being added to the KeptnTask names) label filtering prevents Kuttl from asserting the conditions against the wrong object